src/lib/y2storage/btrfs_subvolume.rb

```diff
@@ -241,29 +247,16 @@ def former_referenced_limit
     # @return [Boolean]
     def require_default_mount_point?
       return false unless filesystem.root?
-      return false if top_level? || default_btrfs_subvolume? || snapshot?
+      return false if top_level? || default_btrfs_subvolume? || for_snapshots?
 
       parent_subvolume.top_level? || parent_subvolume.prefix?
     end
 
-    # Whether the subvolume is for snapshots
-    #
-    # @return [Boolean]
-    def snapshot?
-      path.match?(/^.snapshots/)
-    end
-
-    # Whether the subvolume is the used as prefix (typically @)
+    # Whether the subvolume is used for snapshots
     #
     # @return [Boolean]
-    def prefix?
-      return false unless filesystem.root?
-
-      spec = Y2Storage::VolumeSpecification.for("/")
-
-      return false unless spec&.btrfs_default_subvolume
-
-      spec.btrfs_default_subvolume == path
+    def for_snapshots?
+      path.start_with?(filesystem.snapshots_root)
```

Comment:
This would return true for `@/.snapshotsisms`, which is wrong.

Reply:
Oh, right!